learning_table_tennis_from_scratch/rl_config.py

```diff
@@ -26,55 +26,76 @@ class PPOConfig:
         "n_epochs",
     )
 
+    _sac_params = (
+        "gamma",
+        "ent_coef",
+        "learning_rate",
+        "batch_size",
+        "gradient_steps",
+        "train_freq",
+        "buffer_size",
+        "learning_starts",
+    )
+
     # Additional parameters
     _additional_params = (
-        "num_timesteps",  # 'total_timesteps' passed to PPO.learn()
+        "num_timesteps",  # 'total_timesteps' passed to RL.learn()
         "save_path",  # If set the model is saved to the given path.
         "num_layers",  # Number of layers in the network
         "num_hidden",  # Size of each layer (all layers have same size)
         "log_path",  # Destination for checkpoints and log files
     )
 
-    __slots__ = _ppo_params + _additional_params
+    __slots_ppo__ = _ppo_params + _additional_params
+    __slots_sac__ = _sac_params + _additional_params
```

Comment:
I think names with double-underscore on both sides should be reserved for built-in Python stuff with special meaning.  So maybe better call them something like `_slots_ppo`/`_slots_sac` to make it clear that it is just a custom variable.